addons/website_event/models/event_event.py

```diff
@@ -102,6 +111,22 @@ def _default_question_ids(self):
                                             domain=[('once_per_order', '=', False)])
 
     def _compute_is_participating(self):
```

Comment:
Can we at least set a depends on this? Would say registration_ids + depends_context uid.